static/app/components/events/interfaces/crashContent/exception/content.tsx

```diff
@@ -178,7 +178,7 @@ export function Content({
         )}
         <StyledPre className="exc-message">
           {meta?.[excIdx]?.value?.[''] && !exc.value ? (
-            <AnnotatedText value={exc.value} meta={meta?.[excIdx]?.value?.['']} />
+            <AnnotatedText value={exceptionValue} meta={meta?.[excIdx]?.value?.['']} />
```

Comment:
I also went back and ensured linkify was being applied to annotated text in the stacktrace.

Reply:
I don't think this matters actually since this is only rendered when `exc.value` is falsey